app/controllers/application_controller.rb

```diff
@@ -1,2 +1,10 @@
 class ApplicationController < ActionController::Base
+    protect_from_forgery with: :null_session
+    before_action :configure_permitted_parameters, if: :devise_controller?
```

Comment:
Do you understand what these line of code do?

Reply:
So i just know protect_from_forgery . Like this is used to detect cross-origin requests . So for Rails 5 there will be an exception by default but if it detects these fraudulent activities and makes the session null . I just went through a doc about this https://marcgg.com/blog/2016/08/22/csrf-rails/